.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
```

Comment:
We would like to keep the current prettier settings. can you remove this file?

Reply:
I didn't see the previous prettier settings in the package.json. Will remove this file then.